rust/main/utils/run-locally/src/main.rs

```diff
@@ -589,23 +589,25 @@ fn relayer_restart_invariants_met() -> eyre::Result<bool> {
     let no_metadata_message_count = *matched_logs
         .get(&line_filters)
         .ok_or_else(|| eyre::eyre!("No logs matched line filters"))?;
```

Comment:
related to testing in general rather than to this code line: did you notice a drop in message delivery latency due to this change? I would expect so since we're no longer blocked by the rate limits, would be good to check by looking at the timestamps of `Operation submitted` logs and seeing if they occur earlier in E2E run now